.github/workflows/tests.yml

```diff
@@ -138,16 +138,6 @@ jobs:
           cask: false
           test-bot: false
 
-      - name: Cache Bundler RubyGems
-        uses: actions/cache@v3
-        with:
-          path: ${{ steps.set-up-homebrew.outputs.gems-path }}
-          key: ${{ runner.os }}-rubygems-${{ steps.set-up-homebrew.outputs.gems-hash }}
-          restore-keys: ${{ runner.os }}-rubygems-
-
-      - name: Install Bundler RubyGems
-        run: brew install-bundler-gems --groups=all
```

Comment:
Why no more caching/installing here (and elsewhere)?

Reply:
`brew audit --skip-style` doesn't seem to require gems, or it at least doesn't request any itself. CI seems happy, but I'll double check there's nothing sneaky happening somewhere.

Reply:
Ok, gotcha.